data/sea/54-zebra.h

```diff
@@ -194,16 +194,15 @@ static int64_t zebra_translate_column_tags
     , void **dst
     , const int64_t *src )
 {
-    if (variant_count <= 2) {
-        fprintf (stderr, "Fatal error: found an enum with %" PRId64 " variants. Icicle does not support Zebra enums with more than two variants.\n");
+    if (variant_count > 2) {
+        fprintf (stderr, "Fatal error: found an enum with %lld variants. Icicle does not support Zebra enums with more than two variants.\n", variant_count);
```

Comment:
yeah it sucks 100% having to use `PRId64` I wish they would add something nicer :(